api/build-tools/json.lua

```diff
@@ -0,0 +1,400 @@
+--
+-- json.lua
+--
+-- Copyright (c) 2020 rxi
```

Comment:
Does this file's license is compatible with Apache License 2.0?

Reply:
fixed, I add this to LICENSE file